salt/modules/mssql.py

```diff
@@ -195,19 +229,28 @@ def role_create(role, owner=None, **kwargs):
 
         salt minion mssql.role_create role=product01 owner=sysdba
     '''
+    if not grants:
```

Comment:
We need to modify the documentation here to show an example of using grants and indicate that if grants are to be passed in, they should be a list.